webside/views.py

```diff
@@ -124,11 +126,6 @@ def join_notes(notes, joiner="</br>"):
     for note_id in notes:
         note = Note.query.get(note_id)
         if note:
-            body += str(joiner) + str("<b>" + note.title + "</b></br>") + str(note.data)
+            body += f"{joiner}<b>{note.title}</b></br>{note.data}"
```

Comment:
To tylko zeby ci pokazac ze jest cos takiego jak f-string i to zawsze woła str() na obiektach w klamerkach `{}`

https://bulldogjob.pl/readme/python-3-poznaj-f-strings